playbooks/config_ntp.yml

```diff
@@ -88,10 +88,17 @@
         state: started
 
     - name: Configure Windows NTP Peers
-      win_shell: w32tm /config /manualpeerlist:"time.windows.com" /reliable:no /update
+      win_shell: w32tm /config /syncfromflags:manual /manualpeerlist:"{{ windows_ntp_servers }}"
```

Comment:
see my comment regarding windows_ntp_servers. You should use the existing variable (ntp_servers).  I know the format is not the one you want but it is easy to reformat using jinja2 filters

Reply:
Done.